spin-up-hub/src/components/PreviewModal.vue

```diff
@@ -62,11 +62,11 @@ export default {
                             </a>
                             <div class="metadata">
                                 <div class="name">Url</div>
-                                <div class="value">{{ modalData.url }}</div>
+                                <div class="value"><a href="{{ modalData.url }}">{{ modalData.url }}</a></div>
                             </div>
                             <div class="metadata">
                                 <div class="name">Submitted by</div>
-                                <div class="value">@{{ modalData.author }}
+                                <div class="value"><a href="https://github.com/{{ modalData.author }}">@{{ modalData.author }}</a>
```

Comment:
I am dubious about sending people to the author's GH page - few people make this a welcoming place.  Send them to the repo or to a user-provided homepage.

(Indeed a bunch of our stuff has the author as `fermyon`, and github.com/fermyon is completely un-useful.)

Reply:
We already have links sending them to the repo. I am happy to make it a non-link again. I do not have any strong feelings about letting people submit their own homepages but that would require some more refactoring so not a part of this PR.

Let me know if you think just returning it to being non-links would be sufficient for the time being. I do recall conversations with people asking for it to be linked to people's profiles as a sign of recognition.

Reply:
Nah, the only real blocker was the capitalisation of the title. :shipit: